Makefile

```diff
@@ -57,18 +65,23 @@ ci_ctx: ## Build CI container with buildah
 .PHONY: run_ctx_pre_commit
 run_ctx_pre_commit: ci_ctx ## Run pre-commit check in a container
 	if [ "x$(BUILD_VENV_CTX)" == 'xyes' ]; then \
-		podman run --rm cfwm:latest make pre_commit_nodeps ; \
+		podman run --rm \
+			-e BASEDIR=$(BASEDIR) \
+			cfwm:latest bash -c "make pre_commit_nodeps BASEDIR=$(BASEDIR)" ; \
```

Comment:
i am confused about that image 
shouldn't be: ci-framework-image:latest ?

Reply:
not locally. Here we're using the locally built image from the `ci_ctx` target

Reply:
(but maybe we may get the same names.... Would it be clearer?)

Reply:
oh yeah, i see that now. I'm not sure if we want the same here too. lgtm in this way